static/app/components/events/autofix/useAutofixSetup.tsx

```diff
@@ -49,9 +46,7 @@ export function useAutofixSetup(
 
   return {
     ...queryData,
-    canStartAutofix: Boolean(
-      queryData.data?.integration.ok && queryData.data?.genAIConsent.ok
-    ),
+    canStartAutofix: Boolean(queryData.data?.integration.ok),
```

Comment:
I'm not sure what the plan is but do we not need to support some sort of org-wide "enabled" setting in place of gen AI consent?

Reply:
yeah you're right i think i'm removing too much, i need a new thing in place